src/test/tasks.yaml

```diff
@@ -42,36 +42,6 @@ tasks:
       - description: Deploy the test resources
         cmd: "uds zarf package deploy build/zarf-package-uds-core-test-apps-*.zst --confirm --no-progress"
 
-      - description: Wait for the admin app to be ready
```

Comment:
Note: for all of these deletions this is mainly time savings + remove some flakiness caused by some of the ones waiting on pods. Zarf/helm will wait on all standard resources (deployments, etc) and also will wait on our Package CRs because we use kstatus conditions on them.